lib/tasks/delete_events.rake

```diff
@@ -0,0 +1,15 @@
+namespace :delete_events do
+  # to manually test this command run "rake delete_events:delete_old"
+  desc "Delete events that are declined and older than 30 days"
```

Comment:
like the events that are declined or rejected by admin, we have to delete them

Reply:
what i meant was that "declined event" sounds like the admin was invited to the event but declined it. "rejected event" sounds more like the opposite of "approved".

Reply:
in this context yes i mean reject

Reply:
then maybe 'rejected' is the better name to use in the code than 'declined'?

Reply:
in our system we have used the declined keyword including in the admin panel. are you suggesting that we change every instance?

Reply:
i mean, if there's no huge effort or risk involved, i believe it's beneficial to use the least-confusion-creating names of variables/functions/etc. but feel free to correct my assumption about the size of the effort

Reply:
It can take time to test everything thoroughly, as changes would require retesting to ensure everything works as expected. Thank you for understanding and resolving the issue.